_includes/manuals/2.0/2.1_quickstart_installation.md

```diff
@@ -65,12 +65,14 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86
 </div>
 
 <div class="quickstart_os quickstart_os_el7">
-  <p>Enable the EPEL (Extra Packages for Enterprise Linux) and the Foreman repos:</p>
+  <p>Enable the EPEL (Extra Packages for Enterprise Linux), SCL(Software Collections) and the Foreman repositories:</p>
 
 {% highlight bash %}
 sudo yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum -y install centos-release-scl-rh
```

Comment:
If you move this below `foreman-release`, it works for all EL-7 based distros since we ship a copy of this. Without it, it doesn't work on Scientific Linux and Oracle Linux which are listed as supported in the dropdown. The `foreman-release-scl` package might be even better since that's what we've always recommended.

Reply:
Changed this one !